clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StoreResourceDefinition.java

```diff
@@ -137,6 +138,8 @@ public ModelNode transformOperation(ModelNode operation) {
                 }
             };
             builder.addRawOperationTransformationOverride(MapOperations.MAP_PUT_DEFINITION.getName(), new SimpleOperationTransformer(removePropertyTransformer));
+
+            builder.getAttributeBuilder().setDiscard(DiscardAttributeChecker.ALWAYS, PROPERTIES);
```

Comment:
Hmm, or perhaps the operation transformer on line 115 is mapping this onto a parameter?